src/components/Footer.jsx

```diff
@@ -7,41 +7,30 @@
  */
 
 import React from 'react';
-import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
+import { useTranslation } from 'react-i18next';
 import styled from '@emotion/styled';
-import { injectT } from '@ndla/i18n';
 import { Footer, FooterText, EditorName } from '@ndla/ui';
-
-import { getLocale } from '../containers/Locale/localeSelectors';
+import { i18n } from '@ndla/i18n';
```

Comment:
I client.jsx henter du i18n fra './i18n', i ConceptPage henter du i18n fra useTranslation, og her henter du i18n fra '@ndla/i18n'. Litt forvirrende at det er samme navn på alle disse. Det hadde kanskje vore lettare å sjå forskjell om det var forskjell på navna?

Reply:
Usikker på om det hadde vore lettare sidan i18n er den samme instansen overalt og det då ikkje er behov for å skilje dei?
Tanken var at `./i18n.ts ` kunne vera eigendefinert i kvar frontend slik at filene med oversetting kan leggast til der osv. I denne fila blir i18n henta frå `@ndla/i18n` og importert i  `Client.tsx` . Den  deretter sendt  til provideren slik at komponentane kan få tak i den gjennom `useTranslation()`. Det er ein feil i Footer då det er meininga at i18n skal hentast med `useTranslation()` her og. 
 
 Men om du fortsatt meinar at eg burde endre navn,  har du forslag til ein bedre løysing? :)

Reply:
Skjønar kva du meinar no! Skal gjera slik at alle komponentar brukar  `@ndla/i18n`.